### PR TITLE
[GraFx Studio] Start, End and Excluded days

### PR DESCRIPTION
**Pull request by BramVerniest**

### CODE REVIEW

File: docs/GraFx-Studio/guides/template-variables/date/index.md [hunk 1]
```diff
@@ -63,6 +63,50 @@ Will work for patterns which output words such as `MMM`, `MMMM`, `cc` and `ccc`.
...
 
+### Start & End Date
+
+Setting a start and / or end date, will limit the choice for end users, to pick a date in that range.
+
+A start and end date can be
+
+- None
+- Specific Date
+- Relative
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/template-variables/date/index.md:74]: Relative date

File: docs/GraFx-Studio/guides/template-variables/date/index.md [hunk 2]
```diff
@@ -63,6 +63,50 @@ Will work for patterns which output words such as `MMM`, `MMMM`, `cc` and `ccc`.
...
+
+![screenshot](date5.png)
+
+#### Specific Date
+
+The possible earliest date, or possible latest date can be made specific. E.g. Headquarters defined that a campaign can be valid within a larger period, but local marketeers can choose the exact start- and end date.
+
+#### Relative
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/template-variables/date/index.md:82]: Relative date

File: docs/GraFx-Studio/guides/template-variables/date/index.md [hunk 3]
```diff
@@ -63,6 +63,50 @@ Will work for patterns which output words such as `MMM`, `MMMM`, `cc` and `ccc`.
...
+
+Setting a number relative to the current date (current = when you open the document), enables a template designer to set a date prior to the current, or x days after the current date.
+
+A negative number, allows to set a date in the past, x days before the current date.
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/template-variables/date/index.md:86]: You could also add an example here, that you can prevent an end user from selecting a date in the past by setting 0 as relative start date.

File: docs/GraFx-Studio/guides/template-variables/date/index.md [hunk 4]
```diff
@@ -63,6 +63,50 @@ Will work for patterns which output words such as `MMM`, `MMMM`, `cc` and `ccc`.
...
+
+Choose what days of the week should be excluded from the selection available for the end user.
+
+![screenshot](date6.png)
+
+### The result
+
+Your settings will define how the date picker is displayed.
+
+In the Template Editor Workspace you'll see a preview
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/template-variables/date/index.md:98]: Template **Designer** workspace

**Review by jeroencosyn — CHANGES_REQUESTED** [review]

File: docs/GraFx-Studio/guides/template-variables/date/index.md (hunk 1 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/guides/template-variables/date/index.md:74]: check

File: docs/GraFx-Studio/guides/template-variables/date/index.md (hunk 2 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/guides/template-variables/date/index.md:82]: Check

File: docs/GraFx-Studio/guides/template-variables/date/index.md [hunk 5]
```diff
@@ -63,6 +63,53 @@ Will work for patterns which output words such as `MMM`, `MMMM`, `cc` and `ccc`.
...
+
+#### Specific Date
+
+The possible earliest date, or possible latest date can be made specific. E.g. Headquarters defined that a campaign can be valid within a larger period, but local marketeers can choose the exact start- and end date.
+
+#### Relative Date
+
+Setting a number relative to the current date (current = when you open the document), enables a template designer to set a date prior to the current, or x days after the current date.
+
+As an example, setting 0 in the "Start" date (Type Relative Date), prevents the end user to pick a date in the past.
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/template-variables/date/index.md:86]: setting 0 in the "Start date"

The property is "Start date", not "Start"

File: docs/GraFx-Studio/guides/template-variables/date/index.md [hunk 6]
```diff
@@ -63,6 +63,53 @@ Will work for patterns which output words such as `MMM`, `MMMM`, `cc` and `ccc`.
...
+
+![screenshot](date10.png)
+
+
+### Exclude days
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/template-variables/date/index.md:91]: Exclude**d** days

**Review by jeroencosyn — APPROVED** [review]

File: docs/GraFx-Studio/guides/template-variables/date/index.md (hunk 5 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/guides/template-variables/date/index.md:86]: check

File: docs/GraFx-Studio/guides/template-variables/date/index.md (hunk 6 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/guides/template-variables/date/index.md:91]: checke(d)

**Pull request merged**